webapp/service/storage/index.js

```diff
@@ -1 +1,2 @@
 export { SidebarActions, useIsSidebarOpened } from './sidebar'
+export { AnalysisActions } from './analysis'
```

Comment:
I'm thinking that both should be exported only from their own folder.
what do u think?

Reply:
I am thinking that maybe it is better do 3 modules inside the analysis folder one for each entity, are you talking about that? Or just put the three entities at the first level?

Reply:
Also in the next pr which is the pr of the steps I have the feeling that I need to extract the actions in that way too

Reply:
no I meant when you import them, you import from ```import {AnalysisActions} from '@webapp/service/storage/analysis' not from storage